### PR TITLE
add ragged batch support for ort backend

### PR DESCRIPTION
**Pull request by askhade**
Adding ragged batch support for onnxruntime backend

### CODE REVIEW

File: src/onnxruntime.cc [hunk 1]
```diff
@@ -1220,12 +1226,33 @@ ModelInstanceState::SetInputTensors(
...
+          uint32_t input_dims_count;
+          RESPOND_AND_SET_NULL_IF_ERROR(
+              &((*responses)[idx]), TRITONBACKEND_InputProperties(
+                                   input, nullptr, nullptr, &input_shape, &input_dims_count,
+                                   nullptr, nullptr));
+
+          batchn_shape[0] += GetElementCount(input_shape, input_dims_count);
+        }
+      }
+      // The shape for the entire input patch, [total_batch_size, ...]
```
Comment (deadeyegoodwin) [inline src/onnxruntime.cc:1249]: ".. entire input batch... "

File: src/onnxruntime.cc [hunk 2]
```diff
@@ -1283,6 +1310,39 @@ ModelInstanceState::SetInputTensors(
     }
   }
 
+  // Process batch input if any
```
Comment (deadeyegoodwin) [inline src/onnxruntime.cc:1313]: Indent? Note that we have a formatting tool that you can run that should normalize all code formatting. https://github.com/triton-inference-server/server/blob/master/CONTRIBUTING.md#coding-convention

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

**Review by GuanLuo — APPROVED** [review]
The change looks good, just need to address the comments from David.

**Comment by askhade** [comment]
FYI - I haven't signed the SLA yet. Waiting on CELA review.

**Review by deadeyegoodwin — DISMISSED** [review]

**Review by deadeyegoodwin — DISMISSED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**